network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/ConfiguredBusImpl.java

```diff
@@ -84,6 +84,38 @@ public Bus setAngle(double angle) {
         return this;
     }
 
+    @Override
+    public double getFictitiousP0() {
+        return getResource().getAttributes().getFictitiousP0();
+    }
+
+    @Override
+    public Bus setFictitiousP0(double p0) {
+        double oldValue = getResource().getAttributes().getFictitiousP0();
+        if (p0 != oldValue) {
+            updateResource(res -> res.getAttributes().setFictitiousP0(p0));
+            String variantId = index.getNetwork().getVariantManager().getWorkingVariantId();
+            index.notifyUpdate(this, "fictitiousP0", variantId, oldValue, p0);
+        }
```

Comment:
Update also the calculated buses, If we want to remove the overrided empty method testSetterGetter in tck test class BusTest ...

Maybe see what is done in PR : https://github.com/powsybl/powsybl-network-store/pull/427
when setting V and Angle on ConfiuredBusImpl ...

Reply:
yes, we have to update both ways